gpkit/keydict.py

```diff
@@ -149,6 +149,11 @@ class KeySet(KeyDict):
     "KeySets are KeyDicts without values, serving only to filter and map keys"
     collapse_arrays = False
 
+    def add(self, *args):
+        "Adds all args to the keyset"
+        for arg in args:
+            self[arg] = None
```

Comment:
same way people add multiple things to a set -- I assume they write a for loop? Or also implement union, which takes an iterable?